src/services/user-management/user.service.ts

```diff
@@ -90,17 +90,13 @@ export class UserService {
         });
     }
 
-    async findOne(
-        id: number,
-        getPermissionOrganisationInfo = false,
-        getPermissionUsersInfo = false
-    ): Promise<User> {
+    async findOne(id: number, getExtendedInformation: boolean = false): Promise<User> {
```

Comment:
The type annotation "boolean" should be removed, since "false" is making it boolean

Reply:
While true, I still prefer the strong typing of having it explicitly set

Reply:
Fair - in vs code (at least in mine) it will show as an error though :-)